lib/Graph/Graph.cpp

```diff
@@ -381,7 +385,7 @@ class ModuleDottyPrinter : public AbstractDottyPrinter {
 public:
   void visitModule(Module *M) {
     for (auto N : M->getConstants()) {
-      dumpNode(N);
+      dumpNode(N, true);
```

Comment:
Why do we need to still use the random string of numbers in this case?

Reply:
Because a module may have multiple functions with the same nodes, however they should not be the same vertices in the dot output.

Reply:
Isn't this handling Constants though? Clearly I'm confused 🤷‍♂ maybe add a comment explaining what's going on?

Reply:
It's a bit intricate, yeah. This block is handling constants, and is setting the Vertex name of the constant so we can add edges to it later. In this path where we're dumping the Module we use a version of the Function that does node uniquing since it might have multiple vertices for the same Node name. Since we're doing uniquing all vertices need to use the same uniquing rule in order for us to create the right edges.

TLDR: use node names for single function dumping, use uniqued names for Module dumping.